assets/js/remotebuzzer-server.js

```diff
@@ -205,7 +205,7 @@ const requestListener = function (req, res) {
         res.end(content);
     }
 
-    const urlObj = new URL(req.url, 'http://localhost/'); //The consctructor requires to input a base url
+    const urlObj = new URL(req.url, 'http://' + config.webserver.ip);
```

Comment:
```
config.remotebuzzer.serverip
```
Own IP or from Remotebuzzer server (might be running on a different device and Photobooth only acts as client)